examples/config_dc2_steps.yaml

```diff
@@ -0,0 +1,121 @@
+# This is an example of the template specification file for Steps used in DC2 campaigns
+
+# Import the common element templates
+- import: "${CM_CONFIGS}/config_standard_elements.yaml"
+# Now define the steps
+- SpecBlock:
+      name: dc2_step1
+      includes: ['step']
+      data:
+          pipeline_yaml: "${DRP_PIPE_DIR}/pipelines/LSSTCam-imSim/DRP-test-med-1.yaml#step1"
+      child_config:
```

Comment:
I see here that the groups are defined as the `child_config` of the step, and all the group information is below.